web/html/src/manager/recurring/recurring-actions-edit.tsx

```diff
@@ -125,6 +128,12 @@ class RecurringActionsEdit extends React.Component<Props, State> {
     }
   };
 
+  getTypes = () => {
+    let types = ["Highstate", "Custom state"];
+    window.isControlNode === "true" && (types = types.concat("Ansible Playbook"));
```

Comment:
Here and elsewhere, would prefer a regular if condition instead since it's more consistent with code elsewhere.

```suggestion
    if (window.isControlNode === "true") {
      types = types.concat("Ansible Playbook");
    }
```